.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -9,5 +12,8 @@ Fixes #
 
 #### Changes proposed in this pull request:
 
--
--
+* Describe how this PR will make our project even more awesome.
```

Comment:
@optimistanoop  No need to add this as Heading **Changes proposed in this pull request** conveys the same message.